compiler/semcall.nim

```diff
@@ -212,7 +210,6 @@ proc resolveOverloads(c: PContext, n, orig: PNode,
       if result.state != csMatch:
         n.sons.delete(1)
         orig.sons.delete(1)
-        excl n.flags, nfExprCall
```

Comment:
Hrm, the Github diff viewer is messing something up here. This line is not removed in the actual code on my hard drive.